pyfolio/perf_attrib.py

```diff
@@ -165,9 +170,17 @@ def perf_attrib(returns,
 
     if len(missing_factor_loadings_index) > 0:
 
+        if len(missing_factor_loadings_index) > 5:
+            missing_dates_displayed = "{}..{}".format(
+                missing_factor_loadings_index[0],
+                missing_factor_loadings_index[-1]
```

Comment:
If we have two missing dates on either end of a large range, this will read as if we're missing the entire range. I'd think we'd want to use this format only if we're actually missing all of those dates. We could also do something like `"Could not find factor loadings for {n} dates (first missing is {YYYY-MM-DD}, last missing is {YYYY-MM-DD})."` cc @abhijeetkalyan

Reply:
I haven't looked at the code, but from the screenshot I thought that we'd see the `...` only when the dates are contiguous.  If that's not the case, then I think the message you posted in the comment above is a good one, where we show the first missing and last missing.

Reply:
fixed to use @yankees714's suggestion, so that we don't have to check whether the missing dates are contiguous or not